Muszilla/src/Controller/uploadCon.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace App\Controller;
+
+/**
+ * Class to handle the uploading of the files to be uploaded by the users.
+ */
+class uploadCon {
```

Comment:
Class name first letter should be capital

Reply:
Changes Done.